updatecli/updatecli.d/plugin-health-scoring.yaml

```diff
@@ -47,6 +47,10 @@ actions:
     scmid: default
     title: Bump `plugin-health-scoring` docker image version to {{ source "latestRelease" }}
     spec:
+      automerged: true
+      reviewers:
```

Comment:
Ping @alecharp same: as per [ttps://www.updatecli.io/docs/plugins/actions/github/#_parameters](https://www.updatecli.io/docs/plugins/actions/github/#_parameters). Should be `assignees`.

Reply:
I didn't want to assigne the pull request, but really have a review request.

Reply:
Oh my bad, I was not focused and misunderstood. Should be working for you though: you have `write` permission in the repository (but not darin). Let me add him in "Read" (https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews#about-pull-request-reviews)